### PR TITLE
[Bug Fix] Improve generate and loading firewall rules

### PR DESCRIPTION
**Pull request by myrevery**
Improve loading script:

1. Optimize check IPSet

2. Can self-healing
Auto recreate the loading script, avoid issue in some special cases, such as netplan.io package is uninstalled or the default interface is renamed, etc. Now just restart iptables in the WebUI to fix it.

Bug Fix:

Fix the core issue of https://github.com/hestiacp/hestiacp/issues/1849

This issue is caused by https://github.com/serghey-rodin/vesta/pull/332, in order to be compatible with OpenVZ, so added check "net.netfilter.nf_conntrack_max" to test whether the nf_conntrack module is loaded.

So it actually broke the test of nf_conntrack module, because it is "passive check", If the nf_conntrack module is not loaded, the test will inevitably fail.

Now first use modprobe load the nf_conntrack module for active check, and then passive check as a fallback.

Reproduce the issue:

When only iptables call nf_conntrack module in the system, run "v-stop-firewall" and then reboot.

Now unless some program call or manual load nf_conntrack module, run "v-update-firewall" will inevitably fail to enable stateful support.

### CODE REVIEW

File: bin/v-stop-firewall [hunk 1]
```diff
@@ -76,33 +76,28 @@ if [ -d "/etc/sysconfig" ]; then
...
+    # Recreate the Hestia iptables rules loading script
+    rm -f $routable $preup
     if dpkg-query -W -f'${Status}' "netplan*" 2>/dev/null | grep -q "ok installed" && [ -d /etc/netplan ] && [ -n "$(ls -A /etc/netplan 2>/dev/null)" ]; then
-        preup="/usr/lib/networkd-dispatcher/routable.d/10-hestia-iptables"
-        if [ ! -e "$preup" ]; then
-            IFS='%'
-            echo '#!/bin/sh' > $preup
-            echo '' >> $preup
-            echo 'if [ "$IFACE" = "'$(ip route list | awk '/default .+/ {print $5}' | uniq)'" ]; then' >> $preup
-            [ -x "$(which ipset)" ] && echo "    ${HESTIA}/bin/v-update-firewall-ipset" >> $preup
```
Comment (jaapmarcus) [inline bin/v-stop-firewall:86]: Any reason why this line is removed? 

 [ -x "$(which ipset)" ] && echo "    ${HESTIA}/bin/v-update-firewall-ipset" >> $preup

File: bin/v-stop-firewall [hunk 2]
```diff
@@ -76,33 +76,28 @@ if [ -d "/etc/sysconfig" ]; then
...
+        echo "exit 0" >> $routable
+        chmod +x $routable
     else
-        preup="/etc/network/if-pre-up.d/hestia-iptables"
-        if [ ! -e "$preup" ]; then
-            IFS='%'
-            echo '#!/bin/sh' > $preup
-            echo '' >> $preup
-            echo 'if [ "$IFACE" = "'$(ip route list | awk '/default .+/ {print $5}' | uniq)'" ]; then' >> $preup
-            [ -x "$(which ipset)" ] && echo "    ${HESTIA}/bin/v-update-firewall-ipset" >> $preup
```
Comment (jaapmarcus) [inline bin/v-stop-firewall:100]: Same as above...

**Review by jaapmarcus — COMMENTED** [review]
See comments above

**Comment by myrevery** [comment]
Actually I added it in https://github.com/hestiacp/hestiacp/pull/1858

I was very busy at the time, so just focused on fix https://github.com/hestiacp/hestiacp/issues/1689, because the loading script at the time couldn't be auto recreate, so added it make sure "v-stop-firewall" and "v-update-firewall" create same loading script.

Now it just revert to correct logic of "v-stop-firewall", that is, when the firewall is disabled (WebUI Hestia plug-in switch), there is no need to update the IPSet, because then iptables will not call IPSet.

**Comment by jaapmarcus** [comment]
You got a point merged we are aiming for a release  next week.

**Pull request merged**